cpu/esp32/log_module.c

```diff
@@ -25,7 +25,7 @@
 #include "log.h"
 #include "syscalls.h"
 
-extern char _printf_buf[PRINTF_BUFSIZ];
+char _printf_buf[PRINTF_BUFSIZ];
```

Comment:
maybe make these static?

Reply:
You are right. `_printf_buf` wasn't static since it was used at different places. But now, where it is used only here, it could be static. But, I would leave it for the moment. I hope that I can remove it 
when I'm revising this module in next days.

Reply:
ok!